views/js/runner/plugins/controls/timer/plugin.js

```diff
@@ -58,20 +58,29 @@ define([
              * @param {Object} config - the current config, especially for the warnings
              * @return {Promise<Object[]>} the list of timers for the current context
              */
-            this.loadTimers = function loadTimers(timeStore, timeConstraints, isLinear,  config){
-                var timers = timersFactory(timeConstraints, isLinear, config);
+            this.loadTimers = function loadTimers(timeStore, testContext, config){
```

Comment:
please don't give the whole testContext as parameter
```js
this.loadTimers = function loadTimers(imeStore, timeConstraints, isLinear, config, extraTime)
```
and please update the function documentation accordingly